src/scripts/main.js

```diff
@@ -1,11 +1,24 @@
 'use strict';
 
 function waitFor(element, eventName) {
-  // write your code here
+  return new Promise((resolve) => {
+    const handler = () => {
+      resolve(
+        `It was ${eventName} on the element: ${element.nodeName}, id: ${element.id}.`,
```

Comment:
The `resolve` function should not have a trailing comma after its argument. This is a syntax error in JavaScript.